### ISSUE TITLE
improve speed

### ISSUE THREAD

**Issue by msubirana** [UBod/msa#16]
Hi,

I am trying to align 1000+ sequences and I saw that the speed is quite low. Is it any way to improve this speed? Parallelization? Thanks!

**Comment by UBod** [comment]
For larger data sets, please use ClustalOmega. (unless you have already done so) If you are using Windows or Linux, our implementation of ClustalOmega should also make use of parallelization via OpenMP (unlike on Mac, where this features is disable due to technical issues).

**Comment by msubirana** [comment]
thanks! I will try with ClustalOmega.

Regarding parallelization, is it possible with any R package like doParallel?

**Comment by UBod** [comment]
ClustalOmega does the parallelization internally using OpenMP (unless you have a Mac). There is no need to use doParallel or anything like that. However, if you wish to split your data set into pieces and align the pieces in parallel, you can do that using doParallel.
